src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -863,13 +870,23 @@ public static object DeepCopyInner(object original)
             {
```

Comment:
@sergeybykov The external serializers are lazily registered. In some instances, we won't know if the type is supported by a serializer until runtime (for instance generic types).  The IsSupportedType method is a potentially more costly call, so after it has been determined that a type is indeed supported by an external serializer, it gets registered (~line 879)